.zenodo.json

```
@@ -0,0 +1,35 @@
+{
+    "creators": [
+      {
+        "affiliation": "pyOpenSci",
+        "name": "Wasser, Leah",
+        "orcid": "0000-0002-8177-6550"
+      },
+      {
+        "affiliation": "",
+        "name": "Nicholson, David",
+        "orcid": "0000-0002-4261-4719"
+      },
+      {
+        "affiliation": "Earth Lab, University of Colorado - Boulder",
+        "name": "Sasso, Ariane",
+        "orcid": ""
+      },
+      {
+        "affiliation": "",
+        "name": "Batisse , Alexandre",
+        "orcid": ""
```

```suggestion
        "name": "Batisse, Alexandre",
        "orcid": "0000-0001-7912-3422"
```

Thanks for the entry; this seems well paid for just a few comments 😁